products/apigee/terraform.yaml

```diff
@@ -0,0 +1,54 @@
+# Copyright 2020 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Provider::Terraform::Config
+overrides: !ruby/object:Overrides::ResourceOverrides
+  Organization: !ruby/object:Overrides::Terraform::ResourceOverride
+    autogen_async: true
+    examples:
+    - !ruby/object:Provider::Terraform::Examples
+      name: "apigee_organization_cloud_basic"
+      skip_test: true
+    - !ruby/object:Provider::Terraform::Examples
+      # This is a more verbose version of the above that creates all
+      # the resources needed for the acceptance test.
+      name: "apigee_organization_cloud_basic_test"
+      primary_resource_id: "org"
+      test_env_vars:
+        org_id: :ORG_ID
+        billing_account: :BILLING_ACCT
+      skip_docs: true
+    - !ruby/object:Provider::Terraform::Examples
+      name: "apigee_organization_cloud_full"
+      skip_test: true
+    - !ruby/object:Provider::Terraform::Examples
+      # This is a more verbose version of the above that creates all
+      # the resources needed for the acceptance test. While all Apigee
+      # resources in this test are in the GA API, we depend on a service
+      # identity resource which is only available in the beta provider.
+      name: "apigee_organization_cloud_full_test"
+      primary_resource_id: "org"
+      test_env_vars:
+        org_id: :ORG_ID
+        billing_account: :BILLING_ACCT
+      skip_docs: true
+      min_version: beta
```

Comment:
MM seems to be generating this example in the GA provider (causing CI build errors), and it isn't clear to me why. The configuration here looks correct. I'm looking to see if there is a potential MM bug

Reply:
I found the issue in the [test template](https://github.com/GoogleCloudPlatform/magic-modules/blob/master/templates/terraform/examples/base_configs/test_file.go.erb#L20). Since Apigee has no beta API, and the template takes the minimum version of the API min_version and not of the example, this generates the test for the GA version of the provider. I'm not fluent enough in Ruby to fix that, but a workaround is to define a beta version of the API with the same v1 endpoint. I'll push a commit with that, but will gladly change if you can find a way to fix the test template instead.

Reply:
I tried to open an issue in the MM repo but I can only open provider specific issues from there.

Reply:
There's not a great place to file this, but I made one in the google provider repo where it will actually get monitored: https://github.com/hashicorp/terraform-provider-google/issues/8174

Thanks for looking into this